docs/usage/java.md

```diff
@@ -65,4 +65,16 @@ Here is an example configuration to work with custom Artifactory servers using a
 }
 ```
 
-In the above config, the custom registry URLs are defined using a package rule, and the username/passwords are set using a host rule each.
+In the above config, the custom registry URLs are defined using a package rule, and the username/passwords are set using a host rule each. If you don't want to store your artifactory credentials in plaintext, you can pass them as an environment variable using a javascript config file like `config.js`:
+
+```js
+module.exports = {
+  hostRules: [
+    {
+      hostType: 'maven',
+      baseUrl: 'https://artifactory.yourcompany.com/',
+      username: process.env.ARTIFACTORY_USR,
+      password: process.env.ARTIFACTORY_PSW
```

Comment:
```suggestion
      username: process.env.ARTIFACTORY_USERNAME,
      password: process.env.ARTIFACTORY_PASSSWORD
```

Can you choose any variable name here? Or are we hard-coded to what's there now?
If we can choose any variable name, please spell out the full name instead of using an abbreviation.
I think having clear variable names is really helpful. 😄